### PR TITLE
docs: instructions on generating SP1 proofs

### LINKED ISSUE #561: docs: generate sp1 proof
Add new section on docs on proof generation. Specifically with instructions on writing sp1 proofs and submitting them to aligned

### PR DESCRIPTION
**Pull request by taturosati**
New section with instructions on generating SP1 proofs an submitting them to aligned

### CODE REVIEW

File: docs/guides/3_generating_proofs.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+## SP1
+
+### Dependencies
+
+This guide assumes that:
+
+- sp1 prover installed (instructions [here](https://succinctlabs.github.io/sp1/getting-started/install.html))
+- sp1 project to generate the proofs
+  (instructions [here](https://succinctlabs.github.io/sp1/generating-proofs/setup.html))
+- aligned installed (instructions [here](../introduction/1_getting_started.md#quickstart))
```
Comment (NicolasRampoldi) [inline docs/guides/3_generating_proofs.md:19]: `"(instructions"` parenthesis are not closed

File: docs/guides/3_generating_proofs.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+### How to get the proof verified by Aligned
+
+After generating the proof, you will have to find two different files:
+
+- proof file: usually found under `script` directory, with the name `proof.json` or similar
+- elf file: usually found under `program/elf/` directory
+
+Then, you can send the proof to the AlignedLayer network by running the following command
```
Comment (NicolasRampoldi) [inline docs/guides/3_generating_proofs.md]: We could change AlignedLayer for Aligned

File: docs/guides/3_generating_proofs.md (hunk 1 above)
Comment (srosati) [reply docs/guides/3_generating_proofs.md:19]: I think they are

File: docs/guides/3_generating_proofs.md (hunk 2 above)
Comment (taturosati) [reply docs/guides/3_generating_proofs.md]: Changed

**Review by NicolasRampoldi — APPROVED** [review]

File: docs/guides/3_generating_proofs.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+# Generating Proofs For Aligned
+
+## Table of contents
+
+- [SP1](#sp1)
```
Comment (JuArce) [inline docs/guides/3_generating_proofs.md]: We should remove the table of content. We don't have it in other pages and gitbook provides a table of content automatically in the right side of the page

File: docs/guides/3_generating_proofs.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+```bash
+cargo run --release
+```
+
+### How to get the proof verified by Aligned
+
+After generating the proof, you will have to find two different files:
+
+- proof file: usually found under `script` directory, with the name `proof.json` or similar
```
Comment (JuArce) [inline docs/guides/3_generating_proofs.md]:
```suggestion
- **proof file**: usually found under `script` directory, with the name `proof.json` or similar
```

File: docs/guides/3_generating_proofs.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+- elf file: usually found under `program/elf/` directory
```
Comment (JuArce) [inline docs/guides/3_generating_proofs.md]:
```suggestion
- **elf file**: usually found under `program/elf/` directory
```

**Review by JuArce — CHANGES_REQUESTED** [review]

File: docs/guides/3_generating_proofs.md (hunk 3 above)
Comment (taturosati) [reply docs/guides/3_generating_proofs.md]: removed

**Review by JuArce — APPROVED** [review]

**Pull request merged**